src/_ctf-private/ctf-sign-up-banner/SignupBanner.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Button, Container, Flex, Text, useBreakpointValue, useTheme } from '@chakra-ui/react';
+import Link from 'next/link';
+
+import { useSticky } from './hooks/useSticky';
+
+import CfLogo from '@icons/cf-logo.svg';
+
+export const SignupBanner = () => {
+  const isMobile = useBreakpointValue({ base: true, md: false });
+  const theme = useTheme();
+  // const MotionButton = motion(Button, { forwardMotionProps: true });
+  const signupLink =
+    'https://www.contentful.com/starter-templates/ecommerce-website/sign-up/?action=create_starter_template&template_name=ecommerce';
+  const systemUIFontFamilies =
+    '"Avenir Next W01", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"';
+
+  const { sticky, stickyRef } = useSticky();
```

Comment:
Did you explore a CSS-first approach to have the banner with `position: sticky`?

Reply:
Ahh true, I thought it has really bad support but I guess that was a while ago. I am a bit behind with the CSS stuff.